hircluster.h

```diff
@@ -113,11 +111,11 @@ typedef struct redisClusterContext {
     char errstr[128]; /* String representation of error when applicable */
 
     /* Configurations */
-    int flags;                                  /* Configuration flags */
-    struct timeval *connect_timeout;            /* TCP connect timeout */
-    struct timeval *command_timeout;            /* Receive and send timeout */
-    int max_retry_count;                        /* Allowed retry attempts */
-    char password[CONFIG_AUTHPASS_MAX_LEN + 1]; /* Include a null terminator */
+    int flags;                       /* Configuration flags */
+    struct timeval *connect_timeout; /* TCP connect timeout */
+    struct timeval *command_timeout; /* Receive and send timeout */
+    int max_retry_count;             /* Allowed retry attempts */
+    char *password;                  /* Authentication password */
```

Comment:
Why move the comment alignment column? (Doesn't matter but I just don't see why.)

Reply:
Its auto-formatted this way (`make format`)